components/portal/main/services.js

```diff
@@ -37,7 +37,6 @@ define(['angular'], function(angular) {
         userPromise = prom
           .then(function(result, status) { // success function
             var person = result.data.person;
-            if (APP_FLAGS.loginOnLoad) {
```

Comment:
Since loginOnLoad is almost always false, instead of removing this if condition, we can just drop lines 40-53